### DISCUSSION TITLE
Why does this project not have a lockfile?

### DISCUSSION THREAD

**Discussion by tom-sherman** [vadimdemedes/ink#654] [Q&A] [answered]
This makes contribution very difficult. I can't run the tests because of prettier failures, for example.

**Comment by sindresorhus** [comment, marked as answer]
Should be fixed now. For background on why, see: https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514

**Reply by tom-sherman** [reply]
Thanks for the answer although I can't say I agree with the reasoning!
